File: src/main/java/edu/spring/td1/models/Category.java
package edu.spring.td1.models;

import java.util.List;

public class Category {

    private String name;
    private List<Element> elements;


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Element> getElements() {
        return elements;
    }

    public void addElement(Element element) {
        this.elements.add(element);
    }

    public void removeElement(Element element) {
        this.elements.remove(element);
    }
}

File: src/main/kotlin/edu/spring/td1/models/Category.kt
package edu.spring.td1.models

import java.util.HashSet

data class Category(var label:String) {
    var items:HashSet<Item> = HashSet()

    fun addItem(item:Item) : Boolean {
        return items.add(item)
    }

    fun removeItem(item:Item) : Boolean {
        return items.remove(item)
    }


}